src/org/zaproxy/zap/extension/exportreport/ExtensionExportReport.java

```diff
@@ -72,15 +73,17 @@
     private ArrayList<String> alertSeverity = new ArrayList<String>();
     private ArrayList<String> alertDetails = new ArrayList<String>();
     private ArrayList<String> alertAdditional = new ArrayList<String>();
-    private int maxList = 0;
-    private int SOURCE_COUNT = 8;
+    private FileList fileList = new FileList();
+    public int maxList = 0;
```

Comment:
Does it really need to be public? (there's already a method exposing its value, `extensionGetMaxList()`)